projects/schematics/jest.ts

```diff
@@ -0,0 +1,21 @@
+Object.defineProperty(window, 'CSS', { value: null });
```

Comment:
Why do we need this properties for non UI tests? Can we add them later if it will be required?

Reply:
Thanks for catching this.
It's a copy-paste left over from other libs (when the idea was to switch the whole lib to jest).